### DISCUSSION TITLE
Ability to change the default tags

### DISCUSSION THREAD

**Discussion by ghost** [mateusz-bak/openreads#728] [Ideas]
An option to change the default tag or to not use one would be great. I don’t own all the books that I add and to delete that tag and change it to one more fit one by one gets tiring.

**Comment by OrionCrim** [comment]
I think this is the same issue I'm experiencing.
After adding a tag to a book, that tag is the default for all books added thereafter. I've added other tags in the past, but that's the only one it keeps defaulting to.
I'd like to revert back to no default - or a configurable default.

**Comment by acbarker19** [comment]
This was actually the result of a change I requested, and I wanted to provide a little more info for anyone who may work on this. I had wanted to track books I own so I can avoid needing to search the shelf before getting a book. Ultimately, it was decided that this feature should act as a tag that can't be deleted, and it was set to default on. See the discussion on #417 for more details. It sounds like there could be room for a setting for the default state similar to the one added as part of #470. I can definitely see how it may not be ideal for others to default to the on state.

**Comment by ikanakova** [comment]
I delete the "owned" tag on every book because I don't have books that I don't own, so this tag is irrelevant to me and it would be really nice if we could choose our own default tag or disable the default tag altogether as I wrote in https://github.com/mateusz-bak/openreads/issues/417#issuecomment-1975218782.

**Comment by mateusz-bak** [comment]
Hi, I will make the default tag optional and the user will specify what is it's name.

**Comment by mateusz-bak** [comment]
Implemented in https://github.com/mateusz-bak/openreads/pull/800
